config/ftbquests/quests/chapters/hidden_quests.snbt

```diff
@@ -192,5 +192,14 @@
 				stage: "lesser_tartaric_gem"
 			}]
 		}
+		{
+			x: -14.0d
+			y: -6.5d
+			id: "2ECD3FE62B90722E"
+			tasks: [{
+				id: "354F1CF4ADCBD54A"
+				type: "checkmark"

```

Comment:
A checkmark task in a hidden quest?

Reply:
oh woops, that was a junk quest I was using to get NBT data